Mower/Mower_Sketch/src/main.cpp

```diff
@@ -167,6 +168,15 @@ void setUpWifi()
     esp_now_register_recv_cb(OnDataRecv);
 }
 
+// Funktion Filtern()                                            
+// Bildet einen Tiefpassfilter (RC-Glied) nach.           
+// FF = Filterfaktor;  Tau = FF / Aufruffrequenz            
+// FiltVal der gefilterte Wert, NewVal der neue gelesene Wert; FF Filterfaktor    
+void Filtern(float &FiltVal, float NewVal, int FF)
+{
+  FiltVal= ((FiltVal * FF) + NewVal) / (FF +1);  
```

Comment:
I guess, instead of doing this filter, you could also increase the number of samples for the checkSensorState function and use another threshold to indicate if the mower is mowing 🤷‍♂️